MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
You're completely right, it should works with _data_files_ and I share your duplications concerns.

But after several unsuccessful tries, I stumble on [this](https://kence.org/2015/03/09/secret-sauce-to-including-data_files-in-python-bdist_rpm) and that's the only workaround I found.